packages/libs/react-components/src/utils/convertStyles.ts

```diff
@@ -0,0 +1,27 @@
+import { Primitive } from '../typings';
+
+export const convertStyles = (
+  styles: Record<string, Primitive | Record<string, Primitive>>,
+): Record<string, Primitive | Record<string, Primitive>> => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const newStyles = {} as any;
+  Object.entries(styles).forEach(([key, value]) => {
+    if (value === undefined) return;
+
+    if (typeof value === 'object') {
+      return Object.keys(value).forEach((mediaQueryKey) => {
+        if (newStyles['@' + mediaQueryKey] === undefined) {
+          newStyles['@' + mediaQueryKey] = {};
+        }
+
+        const prop = styles[key];
+        if (typeof prop === 'object') {
```

Comment:
I'm a bit confused. Isn't `prop` the same as `value` so it would be of type "object" by default?

Reply:
it didnt' but now that you mentioned it, i refactored and it looks nicer now